content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -242,43 +240,40 @@ EOF
 systemctl daemon-reload
 ```
 
-Now `kubelet` is ready to use the specified CRI runtime, and you can continue
-with the `kubeadm init` and `kubeadm join` workflow to deploy Kubernetes cluster.
-
-You may also want to set `--cri-socket` to `kubeadm init` and `kubeadm reset` when
+You must also set `--cri-socket` to `kubeadm init` and `kubeadm reset` when
```

Comment:
same as above. kubeadm have auto-detection, so it is necessary to specify `--cri-socket` only in case the auto-detection does not work (more than one CRI installed on a machine).

Reply:
https://github.com/kubernetes/website/pull/16014#discussion_r319139301